rollup.config.js

```diff
@@ -72,15 +72,10 @@ module.exports = [
     ],
   },
 
-  // CommonJS (for Node) and ES module (for bundlers) build.
-  // (We could have three entries in the configuration array
-  // instead of two, but it's quicker to generate multiple
-  // builds from a single configuration where possible, using
-  // an array for the `output` option, where we can specify
-  // `file` and `format` for each target)
+  // ES module (for bundlers) build.
   {
     input: 'src/index.ts',
-    external: ['cross-fetch', 'cross-fetch/polyfill'],
+    external: ['cross-fetch', 'cross-fetch/polyfill', 'crypto'],
```

Comment:
Since this PR just handles the bundle stuff, we can postpone this change when you will implement the generateTenantToken :D